data/event_scripts.s

```diff
@@ -3595,139 +3595,139 @@ gContestPaintingTough3::
 	.string "The mighty tough\n"
 	.string "hyper POKéMON {STR_VAR_1}$"
 
-gUnknown_0827ECBC:: @ 27ECBC
+gTrickHouse_Mechadoll1_Q1A1:: @ 27ECBC
```

Comment:
Perhaps choose names that describe the contents rather than (or in addition to) the use context?

Reply:
Is there a maximum of characters that I shoud strive for?